database/init_db.go

```diff
@@ -69,7 +69,7 @@ func main() {
 	fmt.Println("table test_objects created in the DB.")
 
 	// load the test data from file
-	testData := "test_objects.sql"
+	testData := "database/test_objects.sql"
```

Comment:
Yes, its easier to execute from the root directoy. If you want to keep this change, please change the DEVELOPMENT.md

Reply:
Updated!